frontend/src/components/Forms/AddPermitForm.js

```diff
@@ -0,0 +1,196 @@
+import React from "react";
+import { connect } from "react-redux";
+import { compose } from "redux";
+import PropTypes from "prop-types";
+import { Field, reduxForm, formValueSelector } from "redux-form";
+import RenderField from "@/components/common/RenderField";
+import RenderSelect from "@/components/common/RenderSelect";
+import RenderDate from "@/components/common/RenderDate";
+import { Form, Button, Col, Row, Popconfirm, Tabs } from "antd";
+import * as FORM from "@/constants/forms";
+import { required, dateNotInFuture } from "@/utils/Validate";
+import { resetForm } from "@/utils/helpers";
+import { getDropdownPermitStatusOptions } from "@/selectors/staticContentSelectors";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  closeModal: PropTypes.func.isRequired,
+  permitStatusOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
+  title: PropTypes.string.isRequired,
+  submitting: PropTypes.bool.isRequired,
+  permitTypeCode: PropTypes.string,
+  permitActivityTypeCode: PropTypes.string,
+};
+
+const defaultProps = {
+  permitTypeCode: "",
+  permitActivityTypeCode: "",
+};
+
+const permitTypes = [
+  {
+    label: "Coal",
+    value: "C",
+  },
+  {
+    label: "Mineral",
+    value: "M",
+  },
+  {
+    label: "Placer",
+    value: "P",
+  },
+  {
+    label: "Sand & Gravel",
+    value: "G",
+  },
+  {
+    label: "Quarry",
+    value: "Q",
+  },
+];
+
+const permitActivityTypes = [
+  {
+    label: "Operation",
+    value: "",
+  },
+  {
+    label: "Exploration",
+    value: "X",
+  },
+];
+
+const selector = formValueSelector(FORM.ADD_PERMIT);
+const { TabPane } = Tabs;
+
+const validate = (values) => {
+  const errors = {};
+  if (
+    values.permit_activity_type === "X" &&
+    !(values.permit_type === "C" || values.permit_type === "M")
+  ) {
+    errors.permit_activity_type = "Exploration activity is only valid for Coal and Placer permits";
```

Comment:
This looks like business logic in the frontend

Reply:
Correct. Unless we're moving to server side validation for form validation rules, this is the only way for a good user experience here.

Reply:
That's fair. Do we have comparable validation for this on the backend as well?

Reply:
As far as the back-end is concerned, the permit is a free-form string. Although we could, it feels like a smell to do string parsing and validate specific characters. This is data entry nice-ity mainly.

Storing things as separate type codes at the data level is actually an item in the backlog and a longer term goal. If or when we do that, applying API validation would make a lot of sense.

Reply:
> "This is data entry nice-ity mainly."
Fair enough. I'll mark this as resolved